ytypes/leaf.go

```diff
@@ -337,7 +339,8 @@ type Binary []byte
 
 // unmarshalLeaf unmarshals a scalar value (determined by json.Unmarshal) into
 // the parent containing the leaf.
-//   schema points to the schema for the leaf type.
+//
+//	schema points to the schema for the leaf type.
```

Comment:
Change to a [list](https://go.dev/doc/comment#lists) instead of the current erroneous indenting.